tools/autograd/gen_python_functions.py

```diff
@@ -33,7 +33,7 @@
     'copy_sparse_to_sparse_', 'copy_',
     'numpy_T',  # this needs to be an attribute in Python, not a function
     'nonzero(_(out|numpy))?',
-    'set_quantizer_',
+    'set_quantizer_', 'q_per_channel_axis',  # return types not supported yet
```

Comment:
why? we probably need this for serializing PerChannelAffine tensors

Reply:
do you mean quantizer here?

Reply:
apparently we don't support returning int[] :-\ I can look into fixing it properly

Reply:
ah, I was just lazy - it was easy enough to patch